acme/acme/client.py

```diff
@@ -155,7 +156,23 @@ def answer_challenge(self, challb, response):
         :raises .UnexpectedUpdate:
 
         """
-        response = self._post(challb.uri, response)
+        # Because sending keyAuthorization in a response challenge is deprecated,
```

Comment:
nit: I'd argue that it's worse than deprecated, it has been completely ripped out of the spec.

Reply:
I will use a little more moderated term in the comment, but that is the idea ^^ Modified.